.dockerignore

```diff
@@ -6,6 +6,7 @@
 !/package-lock.json
 !/src
 !/tsconfig.json
+!/build.js
```

Comment:
Oh, this is still `build.js`, I forgot about that change.

Reply:
So did I... but thankfully, Docker Hub didn't 😅